src/client/components/pages/Courses/__tests__/MeetingModal.test.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  BoundFunction,
+  fireEvent,
+  GetByText,
+  QueryByText,
+  wait,
+  waitForElement,
+} from '@testing-library/react';
+import { strictEqual } from 'assert';
+import { TERM } from 'common/constants';
+import { TermKey } from 'common/constants/term';
+import React from 'react';
+import { SinonStub, stub } from 'sinon';
+import { render } from 'test-utils';
+import { cs50CourseInstance } from 'testData';
+import MeetingModal from '../MeetingModal';
+
+describe('Meeting Modal', function () {
+  describe('rendering', function () {
+    let getByText: BoundFunction<GetByText>;
+    let queryByText: BoundFunction<QueryByText>;
+    let onCloseStub: SinonStub;
+    const dispatchMessage: SinonStub = stub();
+    const meetingTerm = TERM.FALL;
+    const semKey = meetingTerm.toLowerCase() as TermKey;
+    beforeEach(function () {
+      onCloseStub = stub();
+      ({ getByText, queryByText } = render(
+        <MeetingModal
+          isVisible
+          currentCourseInstance={{
+            course: cs50CourseInstance,
+            term: meetingTerm,
+          }}
+          onClose={onCloseStub}
+        />,
+        dispatchMessage
+      ));
+    });
+    describe('On Open Behavior', function () {
+      it('populates the heading with the correct course instance information', function () {
```

Comment:
Shouldn't we have additional tests in here for the edit action to make sure that it populates any faculty notes previously added etc.?

Reply:
The faculty notes haven't been added in yet (that's in my next [ticket 306](https://github.com/seas-computing/course-planner/issues/306)), so these test would be added there.

Reply:
Ah okay, that figures.